### PR TITLE
radare2: update to 5.6.8.

### PR DESCRIPTION
**Pull request by ypsvlq**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/radare2/template [hunk 1]
```diff
@@ -1,19 +1,32 @@
 # Template file for 'radare2'
-# cutter needs to be bumped when radare2 is updated
+# iaito needs to be bumped when radare2 is updated
```
Comment (tibequadorian) [inline srcpkgs/radare2/template:2]: We don't have iaito in our repositories. Did you know about #33608?

Comment (tibequadorian) [reply srcpkgs/radare2/template:2]: And what should happen to cutter?

Comment (ypsvlq) [reply srcpkgs/radare2/template:2]: > We don't have iaito in our repositories. Did you know about #33608?

Yes, iaito currently can't build against r2 so I didn't include it. I left the comment in for the future, can be removed if you'd rather.

> And what should happen to cutter?

Cutter has already been updated and no longer depends on r2 (#35901).

**Comment by ypsvlq** [comment]
Ugh, I'll rebase when I'm home

**Comment by ypsvlq** [comment]
Seems to just be an issue with CI?

**Pull request merged**